package.json

```diff
@@ -8,7 +8,7 @@
     "clean": "git clean -dfqX -- ./node_modules **/{dist,node_modules}/ **/tsconfig*tsbuildinfo",
     "precompile": "node scripts/precompile.mjs",
     "postcompile": "node scripts/postcompile.mjs",
-    "compile": "tsc --build tsconfig.build.json packages/server/tsconfig.cjs.json; x=$?; mv packages/server/src/.packageVersion.ts.original packages/server/src/packageVersion.ts; exit $x",
+    "compile": "tsc --build tsconfig.build.json packages/*/tsconfig.cjs.json; x=$?; mv packages/server/src/.packageVersion.ts.original packages/server/src/packageVersion.ts; exit $x",
```

Comment:
random thought while reviewing: we should throw in a tiny circle job that confirms we don't check in changes to packageVersion.ts

Reply:
https://github.com/apollographql/apollo-server/issues/6665